### PR TITLE
Tcl/Tk: update to 8.6.11; rebuild Tktable to avoid bug

### PR DESCRIPTION
**Pull request by chrstphrchvz**
Closes #9521

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.7 (19H15)
Xcode 12.3 command line tools

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Review by jmroot — APPROVED** [review]
LGTM. CI failures are from a CI configuration issue which is fixed in current master.

**Comment by chrstphrchvz** [comment]
There is an 8.6.11.1 errata release being published to fix a primarily Windows issue, but I can use it instead to avoid livecheck saying it's outdated.

**Comment by kencu** [comment]
is there a reason @jmroot didn't want to commit this?

**Comment by jmroot** [comment]
I thought @chrstphrchvz was going to bump to 8.6.11.1 first. It would also be nice to get a working CI run.

File: x11/tk/Portfile [hunk 1]
```diff
@@ -19,11 +19,13 @@ platforms           darwin
 
 master_sites        sourceforge:project/tcl/Tcl/${version}
 dist_subdir         tcltk
-distname            ${name}${version}-src
+# Errata release 8.6.11.1
+#distname            ${name}${version}-src
+distname            ${name}${version}.1-src
```
Comment (jmroot) [inline x11/tk/Portfile:24]: Is there a reason not to make the actual port version 8.6.11.1?

Comment (chrstphrchvz) [reply x11/tk/Portfile:24]: I don't think there is a reason to use 8.6.11.1 as the tk port version. The corrected Tk distfile is named 8.6.11.1 merely to distinguish from the original Tk 8.6.11 distfile; upstream provided it as an alternative to them correcting the 8.6.11 distfile without renaming it, or them only offering a patch for package managers to apply to the original distfile. The version reported by Tk is still 8.6.11.

`port livecheck tk` detects 8.6.11 as latest version.

Comment (jmroot) [reply x11/tk/Portfile:24]: OK

**Pull request merged**